### PR TITLE
Add a code of conduct

### PR DESCRIPTION
**Pull request by sebmck**

### CODE REVIEW

**Comment by sebmck** [comment]
where's the CLA?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (samccone) [inline CODE_OF_CONDUCT.md]: sam@samx.it plz

**Comment by sebmck** [comment]
Done

**Comment by samccone** [comment]
:+1: thanks

**Pull request merged**